api/cache/index.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import RedisConfig from './redis';
+const Redis = RedisConfig();
+
+export const get = (key: string) => {
+  return Redis.get(key).catch(err => {
+    console.error(err);
+    return null;
+  });
+};
+
+type SetType = {
+  key: string,
+  value: string,
+  expiration: ?number, // seconds
+};
+
+export const set = (input: SetType) => {
```

Comment:
nit: require an explicit expiration, default to not expiring. Redis has both a SET and a SETEX operation and it'd be very confusing to call `set` and it expires by default even though you didn't pass an expiry date and you're thinking you're using SET.